extension/manifest.json

```diff
@@ -12,8 +12,10 @@
   "content_scripts": [ {
     "matches": ["*://*.wikipedia.org/*"],
     "all_frames": true,
-    "js": [ "js/lib/jquery.js", "js/content.js" ]
+    "js": [ "js/lib/jquery.js", "js/contentScript.js" ],
+	"run_at": "document_end"
   } ],
+  "web_accessible_resources": ["js/generated.pageScript.js"],
```

Comment:
This has no spaces between the brackets, not that it matters much.

Reply:
Yeah it also uses spaces instead of tabs; I think that's the common display for `manifest.json` for browser extension, but I'm not 100% sure. I can fix that.